packages/client/hmi-client/src/services/models/stratification-service.ts

```diff
@@ -6,13 +8,8 @@ export async function fetchStratificationResult(modelA: string, modelB: string,
 	if (!modelA || !modelB || !typeModel) {
 		throw new Error('An ID must be provided for each model');
 	}
-	const resp = await fetch(
-		`http://localhost:8888/api/models/stratify/${modelA}/${modelB}/${typeModel}`,
-		{
-			method: 'GET'
-		}
-	);
-	const output = await resp.json();
+	const resp = await API.get(`model-service/models/stratify/${modelA}/${modelB}/${typeModel}`);
+	const output = resp.data;
```

Comment:
It will be nice to handle basic errors. Something a bit like this.

```suggestion
try {
	const response = await API.get(`model-service/models/stratify/${modelA}/${modelB}/${typeModel}`);
	return response.data;
} catch (error) {
  console.error(error);
  return null;
}
```

Reply:
done